pkg/controller/apmserverelasticsearchassociation/apmserverelasticsearchassociation_controller.go

```diff
@@ -135,17 +138,19 @@ func (r *ReconcileApmServerElasticsearchAssociation) onDelete(obj types.Namespac
 // and what is in the ApmServerElasticsearchAssociation.Spec
 func (r *ReconcileApmServerElasticsearchAssociation) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	defer common.LogReconciliationRun(log, request, "as_name", &r.iteration)()
+	tx, ctx := commonapm.NewTransaction(r.Tracer, request.NamespacedName, "apm_association")
```

Comment:
I did not do that because the controller names are quiet long and would look weird in the APM UI. Also the word `controller` is somewhat incorrect for an APM transaction, if anything it should be called something like `*-reconciliation` or so . But I harmonized the naming to follow the controller names minus `-controller`. Let me know if you have concerns with that.